app/javascript/react/components/TrailShowTile.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const TrailShowTile = (props) => {
+    return(
+        <div>
+            <h1>{props.name}</h1>
+            <div>
+                <h3>{props.city}</h3>
+                <p>{props.zip}</p>
+            </div>
```

Comment:
this div (starting on line 7) isnt serzing any purpose yet. Delete until you add a stile